pl_examples/bug_report_model.py

```diff
@@ -53,11 +53,9 @@ def run():
     model = BoringModel()
     trainer = Trainer(
         default_root_dir=os.getcwd(),
-        limit_train_batches=1,
```

Comment:
should this really be changed in the bug report model?

Reply:
nope :) will revert in final version